src/components/editor/Tabs.tsx

```diff
@@ -150,13 +150,7 @@ export class Tab extends PureComponent<TabProps, {}> {
         onClick={this.onClick}
         onDoubleClick={this.onDoubleClick}
       >
-        {icon && <div
-          className="icon"
-          style={{
-            backgroundImage: `url(svg/${icon}.svg)`
-          }}
-        />
-        }
+        <div className={"monaco-icon-label file-icon " + icon} />
```

Comment:
It's ok not to check because the font will resort to the default icon in this case.